contracts/gateway/GatewayBase.sol

```diff
@@ -265,7 +265,7 @@ contract GatewayBase {
             emit GatewayProven(
                 remoteGateway,
                 _blockHeight,
-                storageRoot,
+                provenStorageRoot,
```

Comment:
😱 
I always wonder why compilation did not fail for such scenarios before !!!!!!